test/semantics/doconcurrent06.f90

```diff
@@ -30,6 +25,9 @@ subroutine do_concurrent_test1(n)
   nc4:             do i4=1,n
   nc5:               do concurrent(i5=1:n)
   nc6:                 do i6=1,n
+!ERROR: EXIT must not leave a DO CONCURRENT statement
+!ERROR: EXIT must not leave a DO CONCURRENT statement
+!ERROR: EXIT must not leave a DO CONCURRENT statement
```

Comment:
Repeated three times?

Reply:
For the same reason given for the test ```doconcurrent05.f90```.